frontend/lib/src/hostComm/HostCommunicationManager.tsx

```diff
@@ -90,7 +98,12 @@ export default class HostCommunicationManager {
    */
   public openHostCommunication = (): void => {
     window.addEventListener("message", this.receiveHostMessage)
-    this.sendMessageToHost({ type: "GUEST_READY" })
+    this.sendMessageToHost({
+      type: "GUEST_READY",
+      stHtmlLoad: window.HtmlLoad,
```

Comment:
Recording HTML load time is a good idea but I believe this will be `null` for SiS because we use a separate index.html hosted by GS. We could set `<script> window.HtmlLoad = new Date(); </script>` there so that this message picks it up but we don't need that bit to go through streamlit, we can measure it outside of streamlit.

Let's just include the JS execution start here.